Potager

Chimerik opened the pull request:
Branche potager créée et modifiée pour prendre en compte nouvel accès BDD.

Nécessite encore pas mal de modif.

Chimerik committed:
Corrections Spells ID

Chimerik committed:
Corrections Cantrip Scaler + ajout commande !walk

Chimerik committed:
Modif diverses

Correction d'un bug affectant les pnjs métamorphosés : ceux-ci perdaient tous leurs buffs dès que leur perso était sauvegardé.

Ajout d'un système qui permet de modifier la position et l'orientation d'un placeable avec les touches du clavier

Chimerik committed:
Custom player

En gros, plein de modifs pour pouvoir avoir une nouvelle classe Player qui hérite de NWPlayer et a ses propres propriétés dotnet

rgranger committed:
Merge branch 'master' into chim

Resoud les conflits

rgranger committed:
Merge branch 'master' into chim

rgranger committed:
chore(git) : run command git add --renormalize .

rgranger committed:
Merge branch 'master' into chim

rgranger committed:
Merge branch 'master' into chim

Chimerik committed:
Auto stash before merge of "chim" and "origin/chim"

Chimerik committed:
Merge remote-tracking branch 'origin/EV_mysql_dapper' into chim

Chimerik committed:
Modif diverses plus potager

Le potager étant à réécrire en mode objet

Chimerik committed:
Modifs potager nouvelle connexion BDD

Et création de branche

rgranger commented:
Je ne suis pas sûre qu'il n'y ai que ton systeme de potager dans cette PR :

![image](https://user-images.githubusercontent.com/5648257/83899406-b42c8a80-a758-11ea-8c9e-30b44ab1de51.png)

Et quand tu marques :

> Nécessite encore pas mal de modif.

Ca donne pas envie de regarder la PR : c'est mieux de faire une PR seulement lorsque le travail est fini. Sauf si tu as des questions et que tu veux qu'on t'aide. Mais dans ce cas il faut que tu mettes des remarques là où tu as des questionnements.

Chimerik committed:
Fin système potager

Abandon du système par uuid qui marche pas comme je pensais.
Retour au système d'id en local int.

Chimerik committed:
Ajout don pour utilisation transformation loup-garou
